Swat/SwatForm.php

```diff
@@ -904,12 +813,12 @@ protected function displayHiddenFields()
             // The character encoding detection field is intentionally not using
             // SwatHtmlTag to avoid minimizing entities.
             echo '<input type="hidden" ',
-                'name="',
-                self::ENCODING_FIELD,
-                '" ',
-                'value="',
-                self::ENCODING_ENTITY_VALUE,
-                '" />';
+            'name="',
```

Comment:
indentation got weird

Reply:
Not sure why php-cs-fixer handles this differently when you use echo with commas instead of periods for concatenation.  Converting to concat and it indents correctly.